.env.example

```diff
@@ -1,4 +1,6 @@
+# IMPORTANT: do not put quotes "" in the URL variables
```

Comment:
If there are quotes, it will complain about the URL not being in the correct format. It thinks the quotes are actually part of the URL.
Maybe I could add some string sanitizing before the vars get used so it can be more robust.